src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int RANDOM_RANGE = 5;
+
+    private Circle getRandomCircle() {
+        Random random = new Random();
```

Comment:
this line is repeated several times in this class, let's make a global variable of it